### ISSUE TITLE
normalization of training data in the range -1 and 1

### ISSUE THREAD

**Issue by cat-policlot** [lucidrains/denoising-diffusion-pytorch#154]
Is it possible to use training data normalized in the range -1 and 1, and if so, how?

**Comment by lucidrains** [comment]
@cat-policlot yea, just set [this](https://github.com/lucidrains/denoising-diffusion-pytorch/blob/main/denoising_diffusion_pytorch/denoising_diffusion_pytorch.py#L454) to `False` in 1.0.5
